esmvaltool/utils/recipe_test_workflow/recipe_test_workflow/app/build_container/rose-app.conf

```diff
@@ -0,0 +1,10 @@
+[command]
+default=singularity build ${CONTAINER_DIR}/${CONTAINER_FILE} ${DOCKER_SOURCE}
```

Comment:
just a quick heads up one would prob best check with JASMIN which `singularity` is best to be used ie the system default one is pretty old, maybe they have a dedicated setup to build singularity containers (that I am not aware of, sadly)

Reply:
Yup, helpdesk were good with using this version of singularity. Singularity has been renamed as apptainer, but this not installed on Jasmin yet. Current installation is doing the job we need it to for now.